components/telemetry-operator/internal/webhook/logpipeline_webhook.go

```diff
@@ -101,6 +106,17 @@ func (v *LogPipelineValidator) Handle(ctx context.Context, req admission.Request
 		}
 	}
 
+	secretsFound := v.validateSecrets(ctx, logPipeline)
+	if !secretsFound {
+		warnMsg := "One or more secrets do not exist, the log pipeline creation would not be completed"
```

Comment:
Let's be more precise about the behaviour in the warn message. I think it might be unclear if the user has to fix this, or it will resolve itself. Better write that applying the configuration will be pending until all secrets are available.